infra-templates/k8s/32/rancher-compose.yml

```diff
@@ -103,6 +103,12 @@
     required: false
     default: rancherlb
     type: string
+  - variable: AUDIT_LOGS
```

Comment:
Can we add this in an appropriate place in the list of questions instead of just adding it to the bottom?

Maybe between pod infra container image and influxdb host path?